examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +73,30 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# NOAA also provides the past 7 days of flare information which we can also parse.
+
+flare_events = Table.read("https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json", format='pandas.json')
```

Comment:
```suggestion
flare_events = pd.read_json("https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json")
```
It's a toss-up between `DataFrame` and `Table` here, but the tiebreaker is that it makes more sense to use `DataFrame` for the other data.